addon/-private/execution_context.js

```diff
@@ -1,18 +1,28 @@
 import { getContext } from './helpers';
+import { getContext as getTestContext } from '@ember/test-helpers'
 import AcceptanceExecutionContext from './execution_context/acceptance';
 import IntegrationExecutionContext from './execution_context/integration';
+import Rfc268Context from './execution_context/rfc268';
 
 const executioncontexts = {
   acceptance: AcceptanceExecutionContext,
-  integration: IntegrationExecutionContext
+  integration: IntegrationExecutionContext,
+  rfc268: Rfc268Context
 };
 
 /*
  * @private
  */
 export function getExecutionContext(pageObjectNode) {
   let testContext = getContext(pageObjectNode);
-  let context = testContext ? 'integration' : 'acceptance';
+  let context;
+  if (testContext) {
+    context = 'integration';
+  } else if (getTestContext()) {
```

Comment:
Could we add some explanatory comments here? Specifically, how does the return value of `getTestContext()` determine whether we should use the `rfc268` context?